src/_includes/modules/hero.njk

```diff
@@ -9,21 +9,39 @@
   {%- if not heroClass -%}
     {%- set heroClass = "hero" -%}
   {%- endif -%}
-  <article class="{{ heroClass }}" id="{{ hero.slug }}">
+  {%- if img_preload.css -%}
```

Comment:
I think rather than hard coding the css into the hero markup, we can have the link tags appear in the document head sort of like this:

https://github.com/gymnasium/gym-eleventy/blob/develop/src/pages/collections/collection.njk#L27-L42

We would need to edit the home page index to include this css file or something. Needs a little bit of work but I think we can make it elegant.

Reply:
If it's too much hassle we can always refactor later.